libgraph/include/katana/PropertyGraph.h

```diff
@@ -346,6 +350,8 @@ class KATANA_EXPORT PropertyGraph {
 
   uint32_t partition_id() const { return rdg_.partition_id(); }
 
+  uint32_t partition_policy_id_() { return rdg_.part_metadata().policy_id_; }
```

Comment:
Drop suffix `_` in the function name. 

nit: Not in favor of having accessor names that differ from the variable names. Either change the variable name or add a TODO to do so later.

Reply:
Didn't mean to have the `_` suffix. I think `partition_policy_id` or `PartitionPolicyID` is the right name though, what name do you propose?

Reply:
Yes, that's fine. I meant the variable name needs to be changed from `policy_id_` to `partition_policy_id_`

Reply:
I think that `policy_id_` in the `PartitionMetadata` class pretty clearly refers to the partition policy id, so I am not sure I agree with the more verbose name change

Reply:
Ok, this is fine.